client/my-sites/upgrades/domain-management/contacts-privacy/card.jsx

```diff
@@ -54,10 +55,18 @@ const ContactsPrivacyCard = React.createClass( {
 	},
 
 	getNotice() {
+		if ( ! this.props.privacyAvailable ) {
```

Comment:
Maybe let's just skip the notice in this case? It's confusing - is the privacy protection disabled? Or is it not available at all and why?